nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/test/resources/core-site.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
```

Comment:
I only did this because I noticed the IDE said that "XML Declaration should proceed all documenting content", so it seemed like xml-stylesheet is another declaration like <?xml version> and should come before anything else.

Technically I don't think it stops anything from working to put it back how it was, but just seemed like an easy thing to change if tools are going to recognize it as being invalid.